.travis.yml

```diff
@@ -7,7 +7,8 @@
         "rvm install 2.2",
         "rvm use 2.2 --fuzzy",
         "export GEMDIR=$(rvm gemdir)",
-        "echo \"ruby=$(basename $GEMDIR) jekyll=$JEKYLL_VERSION\" > pseudo_Gemfile.lock"
+        "echo \"ruby=$(basename $GEMDIR) jekyll=$JEKYLL_VERSION\" > pseudo_Gemfile.lock",
+        "npm install -g npm@3.10.7"
```

Comment:
I don't like that approach, we should just get rid of custom caching and stuff like I did with Bootstrap.

https://github.com/twbs/bootstrap/commit/4265f868ac0e9a7686df71aefc0cc64a16afbe67

And it seems I already have https://github.com/twbs/bootlint/pull/363 so I'm gonna look at it when I have some time.